tests/test_fft.py

```diff
@@ -964,3 +965,182 @@ def test_fft_validate_out(self):
         a = dpnp.ones((10,), dtype=dpnp.float32)
         out = dpnp.empty((10,), dtype=dpnp.complex64)
         assert_raises(ValueError, dpnp.fft.rfft, a, out=out)
+
+
+class TestRfft2:
+    def setup_method(self):
+        numpy.random.seed(42)
+
+    # TODO: add other axes when mkl_fft gh-119 is addressed
+    @pytest.mark.parametrize("dtype", get_all_dtypes(no_complex=True))
+    @pytest.mark.parametrize("axes", [(0, 1)])  # (1, 2),(0, 2),(2, 1),(2, 0)
+    @pytest.mark.parametrize("norm", ["forward", "backward", "ortho"])
+    @pytest.mark.parametrize("order", ["C", "F"])
+    def test_rfft2(self, dtype, axes, norm, order):
+        x1 = numpy.random.uniform(-10, 10, 24)
+        a_np = numpy.array(x1, dtype=dtype).reshape(2, 3, 4, order=order)
+        a = dpnp.asarray(a_np)
+
+        result = dpnp.fft.rfft2(a, axes=axes, norm=norm)
+        expected = numpy.fft.rfft2(a_np, axes=axes, norm=norm)
+        assert_dtype_allclose(result, expected, check_only_type_kind=True)
+
+    # TODO: change to get_all_dtypes() when mkl_fft gh-116 and gh-119 is addressed
```

Comment:
I believe we should exclude `dtype=None` from the loop
```suggestion
    # TODO: change to get_all_dtypes(no_none=True) when mkl_fft gh-116 and gh-119 is addressed
```

Reply:
Yep, it is mainly helpful in test cases with `dtype` input keyword.